conjure-java-core/src/integrationInput/java/com/palantir/product/BinaryAliasExample.java

```diff
@@ -2,21 +2,21 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonValue;
-import java.nio.ByteBuffer;
+import com.palantir.conjure.java.lib.Bytes;
 import java.util.Objects;
 import javax.annotation.Generated;
 
 @Generated("com.palantir.conjure.java.types.AliasGenerator")
 public final class BinaryAliasExample {
-    private final ByteBuffer value;
+    private final Bytes value;
 
-    private BinaryAliasExample(ByteBuffer value) {
+    private BinaryAliasExample(Bytes value) {
         Objects.requireNonNull(value, "value cannot be null");
         this.value = value;
     }
 
     @JsonValue
-    public ByteBuffer get() {
+    public Bytes get() {
```

Comment:
Just flagging that this is a major break and any conjure-java users who have `binary` in their API will impose a compilation break on all their downstream consumers.

Can we prefix the PR with `[break]` so that it's clear this should be a major rev? (Alternatively, perhaps there should be some kind of preview flag?)